client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/PutStreamOperation.java

```diff
@@ -45,7 +45,7 @@ public PutStreamOperation(Codec codec, ChannelFactory channelFactory,
                              long lifespan, TimeUnit lifespanTimeUnit, long maxIdle, TimeUnit maxIdleTimeUnit,
                              ClientStatistics clientStatistics) {
       super(PUT_STREAM_REQUEST, PUT_STREAM_RESPONSE, codec, channelFactory, key, keyBytes, cacheName, topologyId,
-            flags, cfg, null, clientStatistics);
+            flags, cfg, null, clientStatistics, null);
```

Comment:
Why are you not passing the telemetry service here ? This operation is functionally equivalent to `PutOperation`